src/configuration-requestor.spec.ts

```diff
@@ -203,5 +203,13 @@ describe('ConfigurationRequestor', () => {
       await configurationRequestor.fetchAndStoreConfigurations();
       expect(fetchSpy).toHaveBeenCalledTimes(1);
     });
+
+    it('Requests bandits only when model versions are different', async () => {
+      await configurationRequestor.fetchAndStoreConfigurations();
+      expect(fetchSpy).toHaveBeenCalledTimes(2); // Once for UFC, another for bandits
+
+      await configurationRequestor.fetchAndStoreConfigurations();
```

Comment:
Ideally, our test checks that it hasn't changed the scenario as well. You can do this by updating the mock between (this would be my instinct) or changing the response based on call count (like we do in our "retries initial configuration request" tests).

Psudocode:
```
init SDK
expect bandits to have been fetched
fetch (UFC bandit references are not updated)
expect no additional bandit fetches
fetch (bandit references model versions shrunk to a _subset_)
expect no additional bandit fetches
poll for updated ufc (bandit references model versions a _superset_)
expect bandits to have been fetched
```